### DISCUSSION TITLE
Conditional mappings for the current tab?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#7446] [Ideas]
Kitty's new conditional mappings for the focused window are great; however, I need conditional mappings for the focused tab, i.e., mappings for _any_ window within a tab that meets the given criteria.

Would it be hard to implement something like:

```
map --when-in-tab title:mytab kitty_mod+t
```

**Comment by kovidgoyal** [comment]
Not something I am interested in, but PRs are welcome. You should be
able to follow along with the implementation of when_focus_on. Add
another flag like --when-focus-on-tab
